mongo/with_transactions_test.go

```diff
@@ -381,6 +393,24 @@ func TestConvenientTransactions(t *testing.T) {
 		// Assert that transaction is canceled within 500ms and not 2 seconds.
 		assert.Soon(t, callback, 500*time.Millisecond)
 	})
+	t.Run("wrapped transient transaction error retried", func(t *testing.T) {
+		sess, err := client.StartSession()
+		assert.Nil(t, err, "StartSession error: %v", err)
+		defer sess.EndSession(context.Background())
+
+		returnError := true
+		res, err := sess.WithTransaction(context.Background(), func(sessCtx SessionContext) (interface{}, error) {
+			if returnError {
+				returnError = false
+				return nil, wrappedError{CommandError{Name: "test Error", Labels: []string{driver.TransientTransactionError}}}
```

Comment:
nit: split this into multiple lines for readability.

Reply:
done